src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -505,7 +507,27 @@ private static bool GetTls13Support()
             else if (IsOpenSslSupported)
             {
                 // Covers Linux, FreeBSD, illumos and Solaris
-                return OpenSslVersion >= new Version(1,1,1);
+                return OpenSslVersion >= new Version(1, 1, 1);
+            }
+
+            return false;
+        }
+
+        private static bool GetTlsHandshakeCAListSupport()
+        {
+            if (IsOpenSslSupported || IsOSX)
+            {
+                return true;
+            }
+
+            if (IsWindows)
+            {
+                // Sending TrustedIssuers is conditioned on the registry.
```

Comment:
I think Windows7 sends the list by default. e.g. even without registry. There are some test that condition on Windows7 but they really should use this.

Reply:
I use different default values depending if we are running on Windows7 or not, so we should be okay here, unless sending cert issuer list can't be disabled on Windows7 using the registry key below.